Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -530,6 +722,8 @@ - (BOOL)isPictureInPicturePossible
 {
 #if PLATFORM(WATCHOS)
     return NO;
+#elif HAVE(PIP_CONTROLLER)
+    return [_pip isPictureInPicturePossible];
 #else
     return _avPlayerViewController.get().isPictureInPicturePossible;
```

Comment:
some consistency in style would be better, prefer `return [_avPlayerViewController isPictureInPicturePossible]; `